### PR TITLE
Fixup auth

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: ci/azure-release.yml [hunk 1]
```diff
@@ -1,13 +1,10 @@
 jobs:
   - job: ${{ parameters.name }}
     displayName: ${{ parameters.displayName }}
-    condition: and(succeeded(), contains(variables['Build.SourceBranch'], '${{parameters.branchref}}'))
+    # condition: and(succeeded(), contains(variables['Build.SourceBranch'], '${{parameters.branchref}}'))
```
Comment (johnsonw) [inline ci/azure-release.yml]: condition not needed?

Comment (jgrund) [reply ci/azure-release.yml]: Condition is needed, commented out for testing.

File: ci/azure-release.yml [hunk 2]
```diff
@@ -3,20 +3,10 @@ jobs:
     displayName: ${{ parameters.displayName }}
     condition: and(succeeded(), contains(variables['Build.SourceBranch'], '${{parameters.branchref}}'))
     steps:
-      - task: Docker@1
-        displayName: ${{ parameters.displayName }}
+      - script: |
+          docker run -i -e OWNER="managerforlustre" -e PROD="${{parameters.release}}" -e PROJECT="${{parameters.project}}" -e PACKAGE="${{parameters.package}}" -e SPEC="${{parameters.spec}}" -e KEY="$(KEY)" -e IV="$(IV)" -v $(pwd):/build:rw imlteam/copr
```
Comment (johnsonw) [inline ci/azure-release.yml]: Is it possible to add:
```
-e CARGO_HOME=/build/.cargo
```
?

Comment (jgrund) [reply ci/azure-release.yml]: It's possible, but it won't buy as anything until azure has caching.

File: ci/azure-release.yml [hunk 3]
```diff
@@ -3,20 +3,10 @@ jobs:
...
-          volumes: $(Build.SourcesDirectory):/build:rw
-          runInBackground: false
-          envVars: OWNER=managerforlustre
-            PROJECT=${{parameters.project}}
-            PACKAGE=${{parameters.package}}
-            SPEC=${{parameters.spec}}
-            RELEASE=${{parameters.release}}
-            KEY=$key
-            IV=$iv
+          KEY: (copr_key)
```
Comment (utopiabound) [inline ci/azure-release.yml]: Should this be `$(copr_key)` ?

Comment (jgrund) [reply ci/azure-release.yml]: ...yes. Interestingly the publish task is working

**Pull request merged**